src/Resources/public/Admin.js

```diff
@@ -322,7 +322,25 @@ var Admin = {
         });
 
         jQuery('.sonata-filter-form', subject).on('submit', function () {
-            jQuery(this).find('[sonata-filter="true"]:hidden :input').val('');
+            var $form = jQuery(this);
+            $form.find('[sonata-filter="true"]:hidden :input').val('');
+
+            var defaultValues = $.param({'filter': JSON.parse(this.dataset.defaultValues)}).split('&'),
+                submittedValues = $form.serialize().split('&');
+
+            // Compare default and submitted filter values in keyValue representation. (keyValue ex: filter[publish][value][end]=2020-12-12)
+            // Keep submit only than values: non default & non empty values because empty values equals not present.
```

Comment:
What do you mean with "Keep submit only than values"?

Reply:
next values. My bad

Reply:
`Allow submit next valus`